### PR TITLE
QS:Uyuni - first draft

### PR DESCRIPTION
**Pull request by Loquacity**
# Description

First edit of QS content from source material. Mostly consolidating into fewer pages, merging similar content from multiple sources and tidying up for consistency.

Still to do (in a new PR, probably): 

- [ ] Enable book for Uyuni product only 
- [ ] Deprecate duplicated content from original sources.

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [ ] Manager-4.1
- [ ] Manager-4.0

# Links

Partial: https://github.com/SUSE/spacewalk/issues/13040

### CODE REVIEW

**Comment by Loquacity** [comment]
Server done, proxy still to come.

File: modules/quickstart-uyuni/pages/install-uyuni-server.adoc [hunk 1]
```diff
@@ -111,7 +111,6 @@ Do not share the main administration access details.
 .Procedure: Setting Up the Main Administration Account
 . In your web browser, enter the address for the {productname} {webui}.
   This address was provided after you completed setup.
-  For more information, see xref:quickstart-uyuni:setup-uyuni-server.adoc#server-setup-yast[].
 . Log in to the {webui}, navigate to the menu:Create Organization[Organization Name] field, and enter your organization name.
```
Comment (keichwa) [inline modules/quickstart-uyuni/pages/install-uyuni-server.adoc:114]:
```suggestion
. Sign in to the {webui}, navigate to the menu:Create Organization[Organization Name] field, and enter your organization name.
```
(the webui has the term "sign in")

File: modules/quickstart-uyuni/pages/install-uyuni-server.adoc [hunk 2]
```diff
@@ -111,7 +111,6 @@ Do not share the main administration access details.
...
 . In the menu:Create Organization[Desired Login] and menu:Create Organization[Desired Password] fields, enter your username and password.
 . Fill in the Account Information fields including an email for system notifications.
```
Comment (keichwa) [inline modules/quickstart-uyuni/pages/install-uyuni-server.adoc:116]: Wondering whether we want to keep this capitalized?  I think it is the title of the dialog.  If yes, we want adoc markup, probably [guimenu].

**Review by keichwa — APPROVED** [review]

**Review by jcayouette — APPROVED** [review]

File: modules/quickstart-uyuni/pages/install-uyuni-server.adoc [hunk 3]
```diff
@@ -1,92 +1,172 @@
...
+Do not share the main administration access details.
+
+
+
+.Procedure: Setting Up the Main Administration Account
+. In your web browser, enter the address for the {productname} {webui}.
+  This address was provided after you completed setup.
+. Log in to the {webui}, navigate to the menu:Create Organization[Organization Name] field, and enter your organization name.
+. In the menu:Create Organization[Desired Login] and menu:Create Organization[Desired Password] fields, enter your username and password.
+. Fill in the Account Information fields including an email for system notifications.
```
Comment (Loquacity) [inline modules/quickstart-uyuni/pages/install-uyuni-server.adoc:115]:
```suggestion
. Fill in the account information fields, including an email for system notifications.
```

File: modules/quickstart-uyuni/pages/install-uyuni-proxy.adoc [hunk 4]
```diff
@@ -1,18 +1,26 @@
 [[install-proxy-uyuni]]
 = Install {productname} Proxy with {opensuse} Leap
 
-{productname} Proxy can be installed on {opensuse} Leap {opensuse-version}.
+{productname} Proxy can be installed on openSUSE Leap.
+The proxy is installed in the same was as a client, but is designated as a proxy server during installation.
```
Comment (keichwa) [inline modules/quickstart-uyuni/pages/install-uyuni-proxy.adoc:5]:
```suggestion
The proxy is installed in the same way as a client, but is designated as a proxy server during installation.
```

File: modules/quickstart-uyuni/pages/install-uyuni-proxy.adoc [hunk 5]
```diff
@@ -1,18 +1,26 @@
...
-+
+You can use a physical or virtual machine running openSUSE Leap to install the Uyuni Proxy.
+Configure a resolvable fully-qualified domain name on the proxy before you begin, to ensure that it is accessible across the network.
+
+The {productname} Proxy software is available from download.opensuse.org, and you can use zypper to retrieve the software and install it.
 
+
+
+.Procedure: Installing openSUSE Leap with Uyuni Proxy
+. Install openSUSE Leap as the base system, and ensure all available service packs and package updates have been applied.
```
Comment (keichwa) [inline modules/quickstart-uyuni/pages/install-uyuni-proxy.adoc:21]:
```suggestion
. Install openSUSE Leap as the base system, and ensure all package updates have been applied.
```
There are no dedicated Leap SPs ;)

File: modules/quickstart-uyuni/pages/install-uyuni-proxy.adoc [hunk 6]
```diff
@@ -21,27 +29,268 @@ repo=repositories/systemsmanagement:/
...
+====
+It is possible to arrange Salt proxies in a chain.
+In this case, the upstream proxy is named `parent`.
+====
+
+Make sure the TCP ports `4505` and `4506` are open on the proxy.
+The proxy must be able to reach the {productname} Server or a parent proxy on these ports.
+
+The proxy shares some SSL information with the {productname} Server.
+You need to copy the certificate and its key from the {productname} Server or the parent proxy, to the proxy you are setting up.
```
Comment (keichwa) [inline modules/quickstart-uyuni/pages/install-uyuni-proxy.adoc:66]: I'm not sure about the comma here.

**Review by keichwa — APPROVED** [review]

File: modules/quickstart-uyuni/pages/install-uyuni-proxy.adoc (hunk 6 above)
Comment (Loquacity) [reply modules/quickstart-uyuni/pages/install-uyuni-proxy.adoc:66]:
```suggestion
You need to copy the certificate and its key from the {productname} Server or the parent proxy to the proxy you are setting up.
```

**Pull request merged**